mobify/mobifyjs
# Do not patch anchor links on Firefox 29 or above

Status: For review

Reviewers: @noahadams 
## Changes
- The anchor patch is no longer necessary on newer firefox browsers. This turns the patching mechanism off for 29 and above.
## How to test-drive this PR
- Go to http://localhost:3000/tests/fixtures/anchor-test.html and run test 1. On older firefox browsers, when you click test 1, there should be _no_ hash in the URL. In newer Firefox browsers, there _should_ be a hash in the URL. Same with non-Firefox browsers.
